### ISSUE TITLE
Worth adding link to CBS VPC requisites ? 

### ISSUE THREAD

**Issue by Dariukuss** [NetAppDocs/occm#828]
Page: [Configure backup for multi-account access in AWS](https://docs.netapp.com/us-en/occm/reference_backup_multi_account_aws.html)

Hey team - I've send a customer to this docs and he come back with some Qs related to "Why is needed" and "Do we need a VPC S3 endpoint on the second VPC" 

I answered both - And checking the docs maybe they are assuming that prerequisites for the CBS to work are met - But I think could be worth adding a link a the bottom of the page to the single VPC CBS config or just replicate the schema as its shown here: 

https://docs.netapp.com/us-en/occm/task_backup_to_s3.html#requirements 

Hope it makes sense !! 

BR ! 

Dario

**Comment by netapp-tonacki** [comment]
Hi Dario,  Thanks for sending your comment.  I'm not 100% clear on what you are asking for here.  

In the topic for [CVO backup to S3](https://docs.netapp.com/us-en/occm/task_backup_to_s3.html#requirements) we say "By default, backups are created using the same account as the one used for your Cloud Volumes ONTAP system. If you want to use a different AWS account for your backups, you must log in to the AWS portal and link the two accounts."

Then we link off to the [topic you mention](https://docs.netapp.com/us-en/occm/reference_backup_multi_account_aws.html). So that's the use case when these extra steps need to be followed. Just for CVO backup to S3 when the user want backups stored in a different AWS account.

Am I not understanding something?  Is there something I could add to clarify this in the docs?

Let me know.  Thanks, Tom

**Comment by Dariukuss** [comment]
Hey Tom ! 

Thanks for answering - Yeah - I understand and see that is all clear if you go that way - This customer was checking the "Multi-Account CBS config" and there is where all the CVO backup to S3 docs are not defined. So, just proposing to add a link like: 

"To configure multi account CBS - Requisites of single account are preserved" and a pointer to the CVO backup to S3 or jus the diagram at the button of the page. To show the S3 endpoint the peering and so. 

But yeah - it is not that hard to follow .. 

BR and thanks !! 

Dario

**Comment by netapp-tonacki** [comment]
Hey Dario,  Sorry for the delay.  You make a good point.  While most users probably won't be navigating the topics in that direction, it is worth adding text describing when these special steps are needed.  I'll add a quick description for both the AWS and the Azure topic about creating backups in different accounts than the source CVO.

Thanks, Tom

**Comment by Dariukuss** [comment]
Thanks man !
